### DISCUSSION TITLE
Proper way to update self-hosted Docker

### DISCUSSION THREAD

**Discussion by gcleaves** [windmill-labs/windmill#2771] [Questions] [answered]
Hello. I can't seem to be able to update my Docker install. I see this message "Update: v1.217.0-3-g0d93986b1 -> v1.219.1" in instance settings.

I've tried following the docs here: https://www.windmill.dev/docs/advanced/self_host#update . I even edited my .env file to look like this: `WM_IMAGE=ghcr.io/windmill-labs/windmill:1.219.1`

Unfortunately, even after `docker compose down` and `docker compose up -d` up again, I still get a message that I'm not on the latest version.

Any tips?

**Comment by rubenfiszel** [comment]
What is the version displayed after the bounce?

**Reply by gcleaves** [reply]
I still see this in the UI after bounce: Update: v1.217.0-3-g0d93986b1 -> v1.219.1

`docker inspect windmill-test-windmill_server-1` shows this: "Image": "ghcr.io/windmill-labs/windmill:1.219.1"

**Reply by gcleaves** [reply]
wait... i think i see my problem

**Comment by gcleaves** [comment, marked as answer]
Resolved, apologies. I had changed the directory of my docker-compose file. That seems to have confused Docker, and me. I am now on the latest version.
